fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -97824,22 +98134,24 @@ creation_date: 2014-11-13T14:33:02Z
 [Term]
 id: FBbt:00110969
 name: larval serotonergic T1 neuron
-def: "Larval serotonergic T1 neuron whose cell body is located close to the ventral midline in the first thoracic segment." [FlyBase:FBrf0048551, FlyBase:FBrf0219731]
+def: "Larval serotonergic thoracic neuron whose cell body is located close to the ventral midline in the first thoracic segment." [FlyBase:FBrf0048551, FlyBase:FBrf0219731]
 subset: cur
 xref: VFB:FBbt_00110969
 intersection_of: FBbt:00110968 ! serotonergic T1 neuron
-intersection_of: part_of FBbt:00001743 ! larval prothoracic segment
+intersection_of: part_of FBbt:00001727 ! larva
+relationship: has_soma_location FBbt:00111030 ! larval prothoracic neuromere
 created_by: mmc46
 creation_date: 2014-11-13T14:33:02Z
 
 [Term]
 id: FBbt:00110970
 name: adult serotonergic T1 neuron
-def: "Adult serotonergic T1 neuron whose cell body is located in ventromedial and posterior in the prothoracic segment." [FlyBase:FBrf0048551]
+def: "Adult serotonergic thoracic neuron whose cell body is located in a ventromedial and posterior position in the prothoracic segment." [FlyBase:FBrf0048551]
 subset: cur
 xref: VFB:FBbt_00110970
 intersection_of: FBbt:00110968 ! serotonergic T1 neuron
-intersection_of: part_of FBbt:00003020 ! adult prothoracic segment
+intersection_of: part_of FBbt:00003004 ! adult
+relationship: has_soma_location FBbt:00110176 ! adult metathoracic neuromere
```

Comment:
Mesothoracic looks like a mistake here.

Reply:
Could also go through and formalise some more using has_soma_location some 'X neuromere' to define class by location.

Reply:
I think there should be a new ticket for general classification regarding VNC neuromeres. 

Classes such as 'A4 neuron' should be defined by soma location and current 'A4 neuron's will need to be reclassified with soma_location rather than part_of. I also think cell body rinds ought to be part_of their respective region to improve this autoclassification.

This is too big a job for now!